### PR TITLE
New fonts

### PR DESCRIPTION
**Pull request by Unfainthful**
Added new fonts, work is checked.
Electron18 font works correctly in screen resolutions
800x600
1024x768
1600x1200

### CODE REVIEW

**Comment by Xottab-DUTY** [comment]
@Unfainthful can this be used in other repositories too?

**Comment by ForserX** [comment]
Через неделю можешь утащить себе.

File: game/gamedata/configs/ui/ui_mm_opt.xml [hunk 1]
```diff
@@ -645,13 +645,13 @@
             <header_1 x="0" y="0" width="220" height="15">
                 <texture a="150">ui_inGame2_servers_list_button</texture>
                 <auto_static x="2" y="0" width="220" height="15">
-                    <text r="170" g="170" b="170" font="arial_14" x="2" y="1" align="l" vert_align="c" complex_mode="0">ui_mm_action</text>
+                    <text r="170" g="170" b="170" font="arial14" x="2" y="1" align="l" vert_align="c" complex_mode="0">ui_mm_action</text>
```
Comment (Xottab-DUTY) [inline game/gamedata/configs/ui/ui_mm_opt.xml]: 🤔 
```
FATAL ERROR
 
[error]Expression    : 0
[error]Function      : CUIXmlInit::InitFont
[error]File          : ui\UIXmlInit.cpp
[error]Line          : 752
[error]Description   : unknown font
[error]Arguments     : arial14
```

Comment (Unfainthful) [reply game/gamedata/configs/ui/ui_mm_opt.xml]: You can use this comit.

Are you apply change in ui_mm_opt_16.xml ?

**Pull request merged**